GearRent/Networking/APIManager.m

```diff
@@ -55,6 +57,58 @@ void fetchListingsWithCoordinates(NSArray<CLLocation *> *polygonCoordinates, voi
     }];
 }
 
+void fetchNearestCity(double lat, double longitude, void(^completion)(NSString *, NSError *error)){
+    @autoreleasepool{
+        NSDictionary *headers = @{ @"X-RapidAPI-Key": @"4ee4de8731msh81f644e471716bbp14ba33jsnae748db48874",
+                               @"X-RapidAPI-Host": @"forward-reverse-geocoding.p.rapidapi.com" };
+        NSString *requestURL = [NSString stringWithFormat:@"https://forward-reverse-geocoding.p.rapidapi.com/v1/reverse?lat=%f&lon=%f&accept-language=en&polygon_threshold=0.0&zoom=10", lat ,longitude];
+        NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString: requestURL]
```

Comment:
I think in this case this should be fine given that we're just adding numbers to the URL's query string. However, in most cases you want to use [NSURLComponents](https://developer.apple.com/documentation/foundation/nsurlcomponents?language=objc). It prevents encoding issues, etc.

Reply:
Will spend time on this after MVP